done/done.py

```diff
@@ -183,3 +201,98 @@ def max_score(self):
         """The maximum raw score of our problem.
         """
         return 1
+
+    # ScorableXBlockMixin methods
+    def set_score(self, score):
```

Comment:
If this method is identical to the method of parent class, it can be skipped. Any objections?

Reply:
Default methods are not implemented

Reply:
Resolved